src/managed/Microsoft.DotNet.PlatformAbstractions/RuntimeEnvironment.cs

```diff
@@ -36,11 +37,7 @@ public static string GetRuntimeIdentifier()
 
         private static string GetRIDArch()
         {
-            if (!string.IsNullOrEmpty(RuntimeArchitecture))
-            {
-                return $"-{RuntimeArchitecture.ToLowerInvariant()}";
-            }
-            return string.Empty;
+            return $"-{RuntimeArchitecture}";
```

Comment:
nit: based on https://github.com/dotnet/corefx/pull/28578#discussion_r177939345, `'-' + RuntimeArchitecture` or `string.Concat("-", RuntimeArchitecture)` would be slightly cheaper here